pkg/api/helpers/download.go

```diff
@@ -0,0 +1,208 @@
+package helpers
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+
+	"github.com/go-openapi/swag"
+	"github.com/treeverse/lakefs/pkg/api/apigen"
+	"github.com/treeverse/lakefs/pkg/uri"
+	"golang.org/x/sync/errgroup"
+)
+
+const (
+	MinDownloadPartSize        int64 = 1024 * 64       // 64KB
+	DefaultDownloadPartSize    int64 = 1024 * 1024 * 8 // 8MB
+	DefaultDownloadConcurrency       = 10
```

Comment:
I think that one can be controlled by the user as well

Reply:
https://github.com/treeverse/lakeFS/issues/7297